proxy/http/HttpTunnel.cc

```diff
@@ -876,7 +875,6 @@ HttpTunnel::producer_run(HttpTunnelProducer *p)
   // Do the IO on the consumers first so
   //  data doesn't disappear out from
   //  under the tunnel
-  ink_release_assert(p->num_consumers > 0);
```

Comment:
why do you remove this assert ?

Reply:
Because when buffering the post at the first time, there is no consumer.